app/views/recipes/index.html.erb

```diff
@@ -29,8 +30,7 @@
       </tr>
     <% end %>
   </tbody>
-</table>
-
+</table> 
 <br>
 
 <%= link_to 'New Recipe', new_recipe_path %>
```

Comment:
Could you make this file identical to the previous version? It might be better not to mark this file as changed since the changes are not meaningful.